content/6_Advanced/MinCostFlow.mdx

```diff
@@ -211,19 +211,105 @@ and
 
 ### Assignment Problem
 
-<IncompleteSection>
+The assignment problem is best understood as a min cost max flow problem. In our formulation,
+we will assign $W$ workers to $J$ jobs, $J \ge W$, and the cost of assigning the
+$i$th job to the $j$th worker is $C_{i,j}$.
 
-Better resources? I honestly just found some that came up in a google search.
+We will create a flow network with a source node $S$, job nodes $J_i$, worker
+nodes $W_j$, and a sink node $E$.
 
-</IncompleteSection>
+We will have the following edges:
+
+- $(S, J_i)$ with $(capacity, cost) = (1, 0)$
+- $(J_i, W_j)$ with $(1, C_{i,j})$
+- $(W_j, E)$ with $(1, 0)$
+
+The answer will be the minimum cost of the max flow of the graph. An example:
```

Comment:
> The answer will be the minimum cost of the max flow of the graph.

do you just mean the max flow?

Reply:
no, it is the min cost of the max flow

Reply:
what does that mean

Reply:
the minimum cost of a max flow of a graph (see https://mirror.codeforces.com/blog/entry/105330)